maquetado/scss/detail.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-flow: column wrap;
+}
+
+.card-description {
+  margin: 18px 6px;
```

Comment:
For only this styles, I don't create a class I think you can refactor your classes to create less.